.github/workflows/config.yml

```diff
@@ -0,0 +1,27 @@
+name: run-trailmap-tests
+on: push
+jobs: 
+  test_py3:
+    runs-on: ubuntu-latest
+    container:
+      image: cyclus/cycamore:latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: pip install -U pytest networkx matplotlib scipy coverage
+      - run: pip install -U more_itertools
+      - run: coverage run --source=./trailmap/ -m pytest
+      - run: coverage lcov
+      - name: Coveralls GitHub Action
+        uses: coverallsapp/github-action@master
+        with:
+          path-to-lcov: ./coverage.lcov
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - run: coverage report
+      - run: coverage html
```

Comment:
@nuclearkatie do we need these lines during CI?

Reply:
Taking a look at the testing logs...
- it may be useful to keep the `report` version so that it can be reviewed in the logs if necessary
- the `html` version writes a file to the temporary folder where the tests are run, but that is immediately removed upon completion of the tests, so it's probably no necessary.

Reply:
Yeah I agree, keep the `report` and drop the `html`!